### PR TITLE
[Feature][Api] Refactor&Add api.controller.QueueV2Controller

### PR DESCRIPTION
**Pull request by tracehh**
## Verify this pull request

This pull request is code cleanup without any test coverage.

*(or)*

This pull request is already covered by existing tests, such as *(please describe tests)*.

(or)

This change added tests and can be verified as follows:

(or)

If your pull request contain incompatible change, you should also add it to `docs/docs/en/guide/upgrede/incompatible.md`

### CODE REVIEW

File: dolphinscheduler-api/src/main/java/org/apache/dolphinscheduler/api/dto/queue/QueueCreateRequest.java [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+ */
+@ApiModel("QUEUE-CREATE")
+@Data
+public class QueueCreateRequest {
+
+    @ApiModelProperty(example = "queue11", required = true)
+    private String queue;
+
+    @ApiModelProperty(example = "queue11", required = true)
+    private String queueName;
```
Comment (SbloodyS) [inline dolphinscheduler-api/src/main/java/org/apache/dolphinscheduler/api/dto/queue/QueueCreateRequest.java]: It's better to distinguish these two examples to indicate the difference between `queue` and `queueName`.

File: dolphinscheduler-api/src/main/java/org/apache/dolphinscheduler/api/dto/queue/QueueUpdateRequest.java [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+@JsonIgnoreProperties(ignoreUnknown = true)
+@JsonInclude(JsonInclude.Include.NON_NULL)
+@Data
+public class QueueUpdateRequest {
+
+    @ApiModelProperty(example = "queue11", required = true)
+    private String queue;
+
+    @ApiModelProperty(example = "queue11", required = true)
+    private String queueName;
```
Comment (SbloodyS) [inline dolphinscheduler-api/src/main/java/org/apache/dolphinscheduler/api/dto/queue/QueueUpdateRequest.java]: Same as `QueueCreateRequest`

File: dolphinscheduler-api/src/main/java/org/apache/dolphinscheduler/api/controller/QueueV2Controller.java [hunk 3]
```diff
@@ -0,0 +1,174 @@
...
+     * @param queueQueryRequest queueQueryRequest
+     * @return queue list
+     */
+    @ApiOperation(value = "queryQueueListPaging", notes = "QUERY_QUEUE_LIST_PAGING_NOTES")
+    @ApiImplicitParams({
+        @ApiImplicitParam(name = "searchVal", value = "SEARCH_VAL", dataType = "String"),
+        @ApiImplicitParam(name = "pageNo", value = "PAGE_NO", required = true, dataType = "Int", example = "1"),
+        @ApiImplicitParam(name = "pageSize", value = "PAGE_SIZE", required = true, dataType = "Int", example = "20")
+    })
+    @GetMapping(consumes = {"application/json"})
```
Comment (SbloodyS) [inline dolphinscheduler-api/src/main/java/org/apache/dolphinscheduler/api/controller/QueueV2Controller.java]:
```suggestion
    @GetMapping()
```

File: dolphinscheduler-api/src/main/java/org/apache/dolphinscheduler/api/controller/QueueV2Controller.java [hunk 4]
```diff
@@ -0,0 +1,174 @@
...
+    /**
+     * query queue list paging
+     *
+     * @param loginUser         login user
+     * @param queueQueryRequest queueQueryRequest
+     * @return queue list
+     */
+    @ApiOperation(value = "queryQueueListPaging", notes = "QUERY_QUEUE_LIST_PAGING_NOTES")
+    @ApiImplicitParams({
+        @ApiImplicitParam(name = "searchVal", value = "SEARCH_VAL", dataType = "String"),
```
Comment (caishunfeng) [inline dolphinscheduler-api/src/main/java/org/apache/dolphinscheduler/api/controller/QueueV2Controller.java]: Should update due to #11574 upgrade.

Comment (tracehh) [reply dolphinscheduler-api/src/main/java/org/apache/dolphinscheduler/api/controller/QueueV2Controller.java]: ok @caishunfeng

**Review by SbloodyS — APPROVED** [review]
LGTM

**Review by caishunfeng — APPROVED** [review]
+1

**Pull request merged**